benchmark/Main.hs

```diff
@@ -1,2 +1,25 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Main where
+
+import Gauge (bench, bgroup, defaultMain, whnf)
+
+import Time (Day, Nanosecond, Second, ns, sec, toUnit, week)
+
+import qualified Data.Time.Units as TU (Day, Nanosecond, Second, Week, convertUnit)
+
 main :: IO ()
-main = undefined
+main = defaultMain
+    [ bgroup "Second to Nanosecond"
+        [ bench "o'clock"    $ whnf (toUnit @Nanosecond . sec) 1
```

Comment:
Could you switch frm `wnhf` to `nf`? `Rational` is a complex data type that has constructor consisting of several fields.